.gitignore

```diff
@@ -8,4 +9,5 @@ cross
 *.swp
 .idea
 *.iml
+*.test
```

Comment:
I'm good with ignoring this file extension, but out of curiosity what is it used for?  VSCode metadata?